doc/getting-started/getting-started.rst

```diff
@@ -142,7 +142,8 @@ To set up the ACRN build environment on the development computer:
 
    .. code-block:: bash
 
-      sudo pip3 install "elementpath==2.5.0" lxml "xmlschema==1.9.2" defusedxml tqdm
+      cd ~/acrn-work/acrn-hypervisor/misc/config_tools
+      sudo pip3 install -r requirements.txt
```

Comment:
We can not make this change because the GSG says to check out the hypervisor v3.0 branch. The requirements.txt file changes you're making aren't on the v3.0 branch and we can't say to use the v3.1 branch yet because it's not ready.